### ISSUE TITLE
The miner does not use the proxy when submitting a dev fee share. 

### ISSUE THREAD

**Issue by N4FA** [develsoftware/GMinerRelease#495]
I encapsulate the share packets with an Encrypted proxy for avoiding censorship.  （with parameter `--proxy`）
But the shares of dev fee are transferred via plaintext. These packets  do not pass through the proxy, meanwhile the connections are not protected by SSL. That  may cause some problems

**Comment by karlestira** [comment]
yes, this is a very serious problem. I had been told that my shares are monitored(and only a few shares monitored so i think this is dev fee) because of non-encrypted dev fee. I need to block dev fee servers too. Can someone share me these IPs or URLs?

**Comment by zhanMM** [comment]
yes, this is a very serious problem
